boards/arm/nrf52_ruuvi/doc/nrf52_ruuvi.rst

```diff
@@ -0,0 +1,169 @@
+.. _nrf52_ruuvi:
+
+nRF52-ruuvi
+##############
+
+Overview
+********
+
+RuuviTag is a generic sensor node, that fits perfectly with many uses.
```

Comment:
This is too much like marketing, how about replacing lines 9-11 with:

    RuuviTag is an advanced battery-operated open-source Bluetooth
    enabled sensor beacon platform capable of sending temperature, humidity, 
    pressure, and motion information.